nodeletctl/pkg/nodeletctl/nodeletctl.go

```diff
@@ -251,11 +283,19 @@ func UpgradeWorkers(clusterCfg *BootstrapConfig, clusterStatus *ClusterStatus) e
 	wg.Wait()
 	return nil
 }
+
 func InitBootstrapConfig() *BootstrapConfig {
+	calicoConfig := CalicoConfig{}
+	calicoConfig.V4BlockSize = 26
+	calicoConfig.V6BlockSize = 122
+	calicoConfig.V4ContainersCidr = "10.20.0.0/22"
+	calicoConfig.V6ContainersCidr = "fd00:101::/116"
```

Comment:
These values seem to be used in multiple places, consider defining a constant someplace for them.